### DISCUSSION TITLE
Billing Spending Limit page not in dark mode

### DISCUSSION THREAD

**Discussion by peaceiris** [community/community#22201] [Accessibility] [answered]
<ol>
<li>Set appearance to <code>dark</code>
</li>
<li>Go to Billing &amp; Plans</li>
<li>Click the <code>Set up a spending limit</code> of GitHub Actions section</li>
<li>The Dark Mode disabled</li>
</ol>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/2/9281d388ef9c4db4fca8fe0d828f62e7f0ac770e.png" data-download-href="/uploads/short-url/kU3RsyDq9asHyL2LyubeFaHgTFk.png?dl=1" title="Screen Shot 2020-12-09 at 3.28.41" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/30958501/181069464-36d42fd8-743c-4619-ab1f-300a1b098f08.png" alt="Screen Shot 2020-12-09 at 3.28.41" data-base62-sha1="kU3RsyDq9asHyL2LyubeFaHgTFk" width="690" height="414" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/2/9281d388ef9c4db4fca8fe0d828f62e7f0ac770e_2_690x414.png, https://aws1.discourse-cdn.com/github/original/3X/9/2/9281d388ef9c4db4fca8fe0d828f62e7f0ac770e.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/2/9281d388ef9c4db4fca8fe0d828f62e7f0ac770e.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/2/9281d388ef9c4db4fca8fe0d828f62e7f0ac770e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-12-09 at 3.28.41</span><span class="informations">785×472 40.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/f/ffc78c26a96fb86775bbb3125cfac19589fd2784.png" data-download-href="/uploads/short-url/AuJhzwPiwCUAhiZJf5Jg9KyZGpm.png?dl=1" title="Screen Shot 2020-12-09 at 3.30.00" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/30958501/181069465-f10192db-32d7-415d-8249-d17bd1309cf1.png" alt="Screen Shot 2020-12-09 at 3.30.00" data-base62-sha1="AuJhzwPiwCUAhiZJf5Jg9KyZGpm" width="690" height="467" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/f/ffc78c26a96fb86775bbb3125cfac19589fd2784_2_690x467.png, https://aws1.discourse-cdn.com/github/original/3X/f/f/ffc78c26a96fb86775bbb3125cfac19589fd2784.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/f/f/ffc78c26a96fb86775bbb3125cfac19589fd2784.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/f/ffc78c26a96fb86775bbb3125cfac19589fd2784_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-12-09 at 3.30.00</span><span class="informations">780×529 49.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by gaknoll** [comment, marked as answer]
<p>Thanks for reporting this <a class="mention" href="/peaceiris">@peaceiris</a>! We aren’t sure how this page ended up not being supported in dark mode but are tracking the issue on our backlog to get resolved</p>

**Comment by Krrishdhaneja** [comment]
<p>I noticed that the dark mode isnt working in Billing section of settings.</p>

**Comment by kboshold** [comment]
<p>There is a secenario where the color theme is lost and you switch back to the light theme as long as you are on the same page. This happens as follows:</p>
<ul>
<li>Enable DarkMode in Appearance</li>
<li>Do the following clickpath:</li>
</ul>
<ol>
<li>Personal Settings</li>
<li>Billing &amp; plans</li>
<li>“GitHub Free” =&gt; Upgrade</li>
<li>Press “Browser Back” / “History Back” button</li>
<li>Color theme is now light again until you leave the personal settings</li>
</ol>

**Comment by Sumonil** [comment]
<p>Hello, when i go on this page, the dark mode is removed.<br>
<a href="https://github.com/settings/billing/spending_limit" rel="noopener nofollow ugc">https://github.com/settings/billing/spending_limit</a></p>

**Comment by gaknoll** [comment]
<p>Hi <a class="mention" href="/sumonil">@sumonil</a> - thanks for catching and sharing this! This is a known issue that we have on our backlog to fix soon.</p>

**Comment by gaknoll** [comment]
<p>Hi <a class="mention" href="/krrishdhaneja">@Krrishdhaneja</a> - thanks for sharing this with us! We are aware of the billing page issue and have it in our backlog to fix soon</p>

**Comment by ToxiKat** [comment]
<p>when in dark mode if we go to settings/billing/spending_limit the dark mode breaks.</p>

**Comment by gaknoll** [comment]
<p>Appreciate you reporting this <a class="mention" href="/mysticlightning2203">@MysticLightning2203</a>! This is a known issue that we hope to get addressed soon :pray:</p>

**Comment by victoryNap** [comment]
<p><a class="mention" href="/gaknoll">@gaknoll</a> it looks like the page linked to the “Upgrade” button on that billing page is also broken. (<a href="https://github.com/account/upgrade" rel="noopener nofollow ugc">https://github.com/account/upgrade</a>)</p>

**Comment by gaknoll** [comment]
<p><a class="mention" href="/victorynap">@victoryNap</a> we had to temporarily opt out that page for the same reason. But the fix is in our backlog as well so we hope to have it resolved soon!</p>

**Comment by akseyh** [comment]
<p>Dark mode is turns to light mode when i go to Billing&amp;plans page and click Spending limit tab.<br>
</p><aside class="onebox twitterstatus">
  <header class="source">
      <a href="https://twitter.com/i/status/1336659057919324160" target="_blank" rel="noopener nofollow ugc">twitter.com</a>
  </header>
  <article class="onebox-body">
    

<h4><a href="https://twitter.com/i/status/1336659057919324160" target="_blank" rel="noopener nofollow ugc">Şems Yılmaz (akseyh)</a></h4>

<div class="tweet"> </div>

<div class="date">
  <a href="https://twitter.com/i/status/1336659057919324160" target="_blank" rel="noopener nofollow ugc">5:08 AM - 9 Dec 2020</a>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by gaknoll** [comment]
<p>Hi <a class="mention" href="/akseyh">@akseyh</a> - thanks for pointing this out! This is a known issue that we have prioritized in our backlog to get fixed soon</p>

**Comment by iSeiryu** [comment]
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/account/billing/history" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:500/500;"><img src="https://user-images.githubusercontent.com/129137/181069477-c186cc91-8ceb-4c14-80c7-8382e822849e.png" class="thumbnail" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png"></div>

<h3><a href="https://github.com/account/billing/history" target="_blank" rel="noopener nofollow ugc">Build software better, together</a></h3>

<p><a href="https://github.com/account/billing/history" target="_blank" rel="noopener nofollow ugc">nt/billing/history</a></p>

  <p><span class="label1">GitHub is where people build software. More than 50 million people use GitHub to discover, fork, and contribute to over 100 million projects.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<br>
This page disables the dark mode too.

**Comment by cehfisher** [comment]
Hi @peaceiris @queenofcorgis - this issue seems to have been resolved. Could you please confirm and close as resolved?
